concepts/sbtc/README.md

```diff
@@ -1,35 +1,53 @@
-# sBTC
+## Introduction to sBTC
 
-To understand sBTC, we first need to understand the current limitations of Bitcoin (BTC).
+sBTC is a SIP-010 token on the Stacks blockchain that represents Bitcoin (BTC) in a 1:1 ratio. It enables Bitcoin holders to participate in DeFi applications and other smart contract functionalities while maintaining a peg to the underlying Bitcoin.
 
-Bitcoin is to date the most secure and decentralized blockchain. While Bitcoin is the largest cryptocurrency by market cap, comparatively few applications exist within the Bitcoin ecosystem. Developers interested in building applications for the Bitcoin community often find it difficult or impossible to implement their logic directly on the Bitcoin chain. Although Bitcoin has a simple scripting system built in, it lacks the expressiveness of many other smart contract languages.
+### Purpose
 
-sBTC is for:
+The primary purpose of sBTC is to bridge Bitcoin to DeFi via the Stacks blockchain, providing Bitcoin holders with access to the rich functionality of smart contracts without sacrificing the security and value of their BTC holdings.
 
-* Bitcoin holders who want to participate in smart contracts.
-* Developers who want to build applications on Bitcoin.
+### Key Benefits
 
-sBTC empowers developers to build applications on Bitcoin by bridging Bitcoin and [Stacks](https://www.stacks.co/). We achieve this by introducing a fungible token (sBTC) on the Stacks blockchain. The token has the following properties:
+1. **Bitcoin Compatibility**: Allows Bitcoin holders to participate in the Stacks ecosystem without selling their BTC.
+2. **DeFi Access**: Enables BTC holders to participate in DeFi.
+3. **Quick Conversions**: Facilitates rapid movement between BTC and sBTC.
+4. **Decentralized Management**: Utilizes a set of signers for maintaining the peg wallet.
+5. **Community Governance**: Involves the community in key decisions, such as selecting the initial signing set.
```

Comment:
Same as above - i'm not sure how accurate/informative needs to be, but it feels like these statements are missing context and introduce questions that aren't being answered. 

the counter argument is that it's not relevant for the doc, and would just add more text to read through vs being shorter and easier to digest.

Reply:
Right now this is a high-level intro, after the SIP is finalized and released I'll be adding more links/info from the SIP to different parts of the doc, keeping it relatively high-level for this initial version